webpack.config.js

```diff
@@ -0,0 +1,97 @@
+const path = require('path'),
```

Comment:
In order to make application development process easier, we can use webpack-dev-server that provides us with live reloading. This should be used for development only. We can add webpack-dev-server script by simply typing in `yarn add webpack-dev-server@2 --dev` command.